doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1197 @@
+       "      <td>LinearRegression()</td>\n",
+       "      <td>281.491492</td>\n",
+       "      <td>/Users/christy/Documents/github_ray_temp/ray/d...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   location_id                          model_name       error  \\\n",
+       "0          145  DecisionTreeRegressor(max_depth=3)  212.416667   \n",
+       "1          166  DecisionTreeRegressor(max_depth=3)  272.837566   \n",
+       "2          152                  LinearRegression()  281.491492   \n",
+       "\n",
+       "                                              logdir  \n",
+       "0  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "1  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "2  /Users/christy/Documents/github_ray_temp/ray/d...  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Keep only 1 model per location_id with minimum error\n",
+    "final_df = results_df.reset_index()\n",
+    "final_df = final_df.loc[final_df.groupby('location_id')['error'].idxmin()].copy()\n",
+    "final_df.sort_values(by=[\"error\"], inplace=True)\n",
+    "final_df.reset_index(inplace=True, drop=True)\n",
+    "print(final_df.dtypes)\n",
+    "final_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4554830-1546-4353-a41e-621bdf46aeab",
+   "metadata": {},
+   "source": [
+    "## Load a model from checkpoint and perform inference  <a class=\"anchor\" id=\"load_checkpoint\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2b218d5-3e64-4b1d-9ad7-5ae3c8a8a78d",
+   "metadata": {},
+   "source": [
+    "> **[Ray AIR Predictors](https://docs.ray.io/en/latest/ray-air/predictors.html) make batch inference easy since they have internal logic to parallelize the inference.**\n",
+    "\n",
+    "However, in this notebook, we will just restore a single scikit-learn model directly from checkpoint, and demonstrate it can be used for inference.  \n",
+    "\n",
+    "Users can easily obtain checkpoint objects from the Tune results object. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "6f5c78fa-38a9-4e8c-bddc-ffbb45aa2d85",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/christy/Documents/github_ray_temp/ray/doc/source/ray-air/examples/my_Tune_logs/batch_tuning/train_model_34a03_00003_3_location=145,model=DecisionTreeRegressor_max_depth_3_2022-11-09_11-17-54/checkpoint_000000\n",
+      "<class 'ray.train.sklearn.sklearn_checkpoint.SklearnCheckpoint'>\n",
+      "<class 'sklearn.tree._classes.DecisionTreeRegressor'>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get a checkpoint path from Ray Tune results. \n",
+    "checkpoint_path = final_df.logdir[0] + \"/checkpoint_000000\"\n",
```

Comment:
we still shouldn't need to be recreating the checkpoints. Let's sync offline?

Reply:
OK, talk to you soon!